### DISCUSSION TITLE
about wrfout and icar

### DISCUSSION THREAD

**Discussion by wwwliye** [NCAR/icar#190] [Q&A]
Hi~ I want to ask the difference between helpers/wrf2icar.sh  and helpers/wrf/wrf2icar.py, are they independent or what contact in them?

**Comment by gutmann** [comment]
Hi wwwliye, the python code is close to 10 years old and does not do everything that the shell script does.  The shell script relies on NCO to process the files, and is probably slower, but it adds features like rotation of the wind field and removal of the bottom layer from metem files that are important. Ideally someone would take those changes and either update the python code, or write a new pythonic version that wraps all of those enhancements together in a faster script!
